src/Microsoft.SqlTools.ServiceLayer/AzureBlob/Contracts/CreateSasRequest.cs

```diff
@@ -0,0 +1,56 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.AzureBlob.Contracts
+{
+    /// <summary>
+    /// Parameters passed for creating shared access signature
+    /// </summary>
+    public class CreateSasParams
+    {
+        /// <summary>
+        /// Connection URI
+        /// </summary>
+        public string OwnerUri { get; set; }
+        /// <summary>
+        /// Blob container URI
+        /// </summary>
+        public string BlobContainerUri { get; set; }
+        /// <summary>
+        /// Blob container key
+        /// </summary>
+        public string BlobContainerKey { get; set; }
+        /// <summary>
+        /// Storage account name
+        /// </summary>
+        public string StorageAccountName { get; set; }
+        /// <summary>
+        /// Shared access signature expiration date
+        /// format: YYYY-MM-DD
```

Comment:
is this UTC? are you sure it doesn't contain time part?

Reply:
@Charles-Gagnon @udeeshagautam I wanted to ask you guys about this. Should I enforce one date format in the RPC? Currently client can use [these formats](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.parse?view=net-6.0).

Reply:
It looks safer to me to split into three arguments - day, month and a year. You never know if client intent will be properly parsed on the other side having so many potential date formats...

Reply:
We control the message being sent and how the date is formatted so that seems unnecessary. 

So you should make sure that this is formatted as expected (I agree using UTC time would probably be best) and then just ensure that it's formatted without regards to the current culture or anything. (you could also do something like use epoch time, but that shouldn't be necessary)